download_openapi.sh

```diff
@@ -0,0 +1,7 @@
+set -a
+source .env
+set +a
+if [ -z ${SWAGGER_CONFIG_PATH+x} ];
+then echo "SWAGGER_CONFIG_PATH is unset";
```

Comment:
Should we throw an error here?

Reply:
I think it's okay. If it's not set, the local `openapi.yaml` will be used.